client/reader/stream/x-post.jsx

```diff
@@ -165,6 +165,10 @@ class CrossPost extends PureComponent {
 		let xpostTitle = post.title;
 		xpostTitle = xpostTitle.replace( /x-post:/i, '' );
 
+		if ( ! xpostTitle ) {
+			xpostTitle = `(${ translate( 'no title' ) })`;
```

Comment:
Hi! I've found a possible matching string that has already been translated **53** times:
`translate( 'No Title' )` *ES Score: 11*
See 2 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/x-post/no-title)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).

Reply:
@bluefuton maybe switch to this translation and add css to force lowercase on display?

Reply:
@blowery makes sense for English, but I'm wary of doing text transforms on other languages: https://github.com/meetup/sassquatch2/issues/144#issuecomment-118958029

Reply:
sure, makes sense. Let's land this as-is and we can switch to Proper Case if it's warranted